### ISSUE TITLE
Metabolic acidosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5873]
**HPO term**
https://hpo.jax.org/app/browse/term/HP:0001942

**Suggested revision of textual definition**

Proposed Definition: Acid accumulation or depletion of base in the body that is not due to a respiratory cause
--

**Comment by pnrobinson** [comment]
@avizrosenberg
@yongqunh  
@beckysteck

I am going to use the following definition to provide more context. Please reopen this issue if necessary.


Def: Metabolic acidosis (MA) is characterized by a fall in blood pH due to a reduction of serum bicarbonate concentration. This can occur as a result of either the accumulation of acids (high anion gap MA) or the loss of bicarbonate from the gastrointestinal tract or the kidney (hyperchloremic MA). By definition, MA is not due to a respirary cause.
Comment: The Henderson–Hasselbalch method defines metabolic acidosis by the presence of an acid–base imbalance associated with a plasma bicarbonate concentration below 20 mmol/L. The association of this imbalance with decreased pH is called acidemia, which is often described as severe when the pH is equal to or below 7.20. Arterial blood gas measurements can be performed in patients with a decreased plasma bicarbonate level so as to eliminate respiratory alkalosis, confirm the diagnosis of metabolic acidosis, and test for mixed acidosis.

PMID: 31418093
PMID: 17936961
